<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorView</class>
 <widget class="QWidget" name="EditorView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="HeaderLayout" stretch="0,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="CommandBar" name="NavBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CommandBar" name="ToolBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CommandBar" name="WinBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="Container">
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
  <action name="Close">
   <property name="icon">
    <iconset resource="editplugin.qrc">
     <normaloff>:/editplugin/close.png</normaloff>:/editplugin/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close Current Editor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="CloseAll">
   <property name="text">
    <string>CloseAll</string>
   </property>
   <property name="toolTip">
    <string>Close All Editors</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save Current Editor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="SaveAll">
   <property name="text">
    <string>SaveAll</string>
   </property>
   <property name="toolTip">
    <string>Save All Editors</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="File">
   <property name="icon">
    <iconset resource="editplugin.qrc">
     <normaloff>:/editplugin/file.png</normaloff>:/editplugin/file.png</iconset>
   </property>
   <property name="text">
    <string>File</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="OpenAsBinary">
   <property name="text">
    <string>Open As Binary</string>
   </property>
   <property name="toolTip">
    <string>Open In Binary Editor</string>
   </property>
  </action>
  <action name="Revert">
   <property name="text">
    <string>Revert</string>
   </property>
   <property name="toolTip">
    <string>Revert Current Editor</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommandBar</class>
   <extends>QToolBar</extends>
   <header>commandbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="editplugin.qrc"/>
 </resources>
 <connections/>
</ui>
